public/auth.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Looks like the formatting is a little off; this can be fixed through VSCode and doing CTRL+K+F

Reply:
That's the way that VSCode's default HTML formatter makes it. (:

Reply:
Ran it through an external HTML formatter. You happier now?

Reply:
very! :D